<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractionWidget</class>
 <widget class="QWidget" name="ExtractionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTask">
     <property name="text">
      <string>Extracting:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCurrentFile">
     <property name="text">
      <string>current file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFrom">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCurrentVolume">
     <property name="text">
      <string>current volume</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTo">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDestinationFolder">
     <property name="text">
      <string>destination folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
